src/lib.rs

```diff
@@ -72,7 +72,7 @@ impl MemFile {
 	/// The close-on-exec flag is set on the created file descriptor.
 	/// If you want to pass it to a child process, you should use [`libc::dup2`] or something similar *after forking*.
 	/// Disabling the close-on-exec flag before forking causes a race condition with other threads.
-	pub fn create(name: &str, options: &CreateOptions) -> std::io::Result<Self> {
+	pub fn create(name: &str, options: CreateOptions) -> std::io::Result<Self> {
 		let file = sys::memfd_create(name, options.as_flags())?;
 		Ok(Self { file })
 	}
```

Comment:
rust convention use `new()` should we keep the GNU conv ?

Could take a CStr to avoid allocation in sys ? This need to be talk in a issue

Reply:
> rust convention use `new()` should we keep the GNU conv ?

`create` is in line with Rust std: https://doc.rust-lang.org/stable/std/fs/struct.File.html#method.create


> Could take a CStr to avoid allocation in sys ? This need to be talk in a issue

`CStr` could be something to investigate, but I also want to keep this convenient to use. We could use a trait that allows either `CStr` or `str` to be passed, but let's move that to a separate issue.